docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -133,7 +133,17 @@ Here are a few examples of permission combinations that satisfy Rancher's needs:
 
 :::
 
-#### 4. Copy Azure Application Data
+#### 4. Allow Public Client Flows
+
+To login from Rancher CLI you will need to allow Public Client Flows.
+
+1. From the navigation pane on, select **Authentication**.
```

Comment:
```suggestion
1. From the left navigation menu, select **Authentication**.
```

Reply:
Thanks @martyav. I just want to highlight that this was copy-pasted from line 104. Should I update also that?